library/zlib.po

```diff
@@ -38,332 +44,490 @@ msgid ""
 msgid ""
 "*strategy* is used to tune the compression algorithm. Possible values are "
 "``Z_DEFAULT_STRATEGY``, ``Z_FILTERED``, and ``Z_HUFFMAN_ONLY``."
 msgstr ""
+"La *stratégie* permet d'ajuster l'algorithme de compression. Les values "
+"existantes sont ``Z_DEFAULT_STRATEGY``, ``Z_FILTERED``, et ``Z_HUFFMAN_ONLY``."
 
 #: ../Doc/library/zlib.rst:102
 msgid ""
-"*zdict* is a predefined compression dictionary. This is a sequence of bytes "
-"(such as a :class:`bytes` object) containing subsequences that are expected "
-"to occur frequently in the data that is to be compressed. Those subsequences "
-"that are expected to be most common should come at the end of the dictionary."
+"*zdict* is a predefined compression dictionary. This is a sequence of bytes (such "
+"as a :class:`bytes` object) containing subsequences that are expected to occur "
+"frequently in the data that is to be compressed. Those subsequences that are "
+"expected to be most common should come at the end of the dictionary."
 msgstr ""
+"L'argument *zdict* est un dictionnaire de compression pré-défini. Il s'agit d'une "
+"séquence d'octets (tel qu'un objet :class:`bytes`) contenant des sous-séquences "
+"attendues régulièrement pour produire les données à compresser. Ces sous-"
+"séquences devraient se trouver en fin de dictionnaire."
 
 #: ../Doc/library/zlib.rst:107
 msgid "Added the *zdict* parameter and keyword argument support."
 msgstr ""
+"Ajout du support de prise en charge du paramètre *zdict* et de sa value en "
+"argument."
 
 #: ../Doc/library/zlib.rst:117
 msgid ""
-"Computes a CRC (Cyclic Redundancy Check) checksum of *data*. The result is "
-"an unsigned 32-bit integer. If *value* is present, it is used as the "
-"starting value of the checksum; otherwise, a default value of 0 is used.  "
-"Passing in *value* allows computing a running checksum over the "
-"concatenation of several inputs.  The algorithm is not cryptographically "
-"strong, and should not be used for authentication or digital signatures.  "
-"Since the algorithm is designed for use as a checksum algorithm, it is not "
-"suitable for use as a general hash algorithm."
-msgstr ""
+"Computes a CRC (Cyclic Redundancy Check) checksum of *data*. The result is an "
+"unsigned 32-bit integer. If *value* is present, it is used as the starting value "
+"of the checksum; otherwise, a default value of 0 is used.  Passing in *value* "
+"allows computing a running checksum over the concatenation of several inputs.  "
+"The algorithm is not cryptographically strong, and should not be used for "
+"authentication or digital signatures.  Since the algorithm is designed for use as "
+"a checksum algorithm, it is not suitable for use as a general hash algorithm."
+msgstr ""
+"Calcule une somme de contrôle CRC (*Cyclic Redundancy Check*) pour l'argument "
+"*data*.  Il revoie un entier 32-bit non-signé.  Si l'argument *value* est "
+"présent, il permettra de définir la value de départ de la somme de contrôle. "
```

Comment:
valeur. OK I got it, you replaced all "valueur" by "value" when I told you *not* to translated an parameter named *value*.

We do not translate parameter names as they come from the library and we do not translate the implementation. But it's not because a word is used once as a parameter name that we should stop translating it everywhere. We still translate the words, we just don't translate the parameter names. Does it make sense?